skimage/transform/tests/test_registration.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from scipy import ndimage as ndi
+
+from skimage import data
+from skimage.transform import registration
+from skimage._shared.testing import (assert_array_equal,
+                                     assert_array_almost_equal)
+
+
+def test_register_affine():
+    reference = data.camera()[::4, ::4]  # speed things up a little
+    forward = np.array([[1.1, 0, 0],
+                        [0  , 1, 0],
+                        [0  , 0, 1]])
+
+    inverse = np.linalg.inv(forward)
+
+    target = ndi.affine_transform(reference, forward)
```

Comment:
why use here the `ndi` function and not a `scikit-image` transform and `warp`?

Reply:
skimage warp transposes x and y and is 2D only, if I've followed the code correctly...